chia/util/streamable.py

```diff
@@ -44,6 +44,34 @@ def __init__(self, message: str, cls: Type[object]):
         )
 
 
+class ParameterMissingError(StreamableError):
+    def __init__(self, cls: type, missing: List[str]):
+        super().__init__(
+            f"{len(missing)} field{'s' if len(missing) > 1 else ''} missing for {cls.__name__}: {', '.join(missing)}"
```

Comment:
I'm on the fence between english correctness and searchability.  Like being able to search a log for `fields missing for` without having to go all regex on it with `fields? missing for`.  But, in most cases you could just search for `ParameterMissingError`, at least when we don't strip it out in our logging...  so not totally sure I want to bank on that.  Anyways, rambling done.  Up to you.